app/views/ingestibles/_form.html.haml

```diff
@@ -73,7 +73,7 @@
           = f.hidden_field :status, value: @ingestible.status
         %h2= t('ingestible.defaults')
         = f.label t(:genre)
-        - genres = [[t(:mixed_genres), 'mixed']]+get_genres.map {|genre| [I18n.t(genre), genre]}
+        - genres = (['mixed'] + Work::GENRES).map { |genre| [textify_genre(genre), genre] }
```

Comment:
I've also added `mixed` value to `genre_values` i18n node.

Sidenote: I've checked content of `Ingestibles` table and we have number of bad values there (when genre is stored in Hebrew). Not sure if we should fix this.

Reply:
thank you for noticing! Must have been an early bug I had fixed without fixing the data. Now manually fixed in the production DB. You can fix (or just randomly assign) in yours.